vsintegration/src/FSharp.ProjectSystem.Base/Project/AssemblyReferenceNode.cs

```diff
@@ -76,7 +76,7 @@ public override string Url
         {
             get
             {
-                return this.myAssemblyPath;
+                return Path.GetFullPath(this.myAssemblyPath);
```

Comment:
This was used by the caching code to determine if two references were the same. `this.myAssemblyPath` is not absolute and so it made comparing them difficult when the rooted-ness was inconsistent within a solution. It's not a bug fix per se.